torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -70,7 +70,7 @@ bool isBroadcasting(Node *node) {
 // emitted as `0` (leading dimensions.)
 std::tuple<bool, at::optional<size_t>> fusibleExpandTo(at::IntList from, at::IntList to) {
   if (from.size() > to.size()) {
-    return std::make_tuple<bool, ssize_t>(false, {});
+    return std::make_tuple<bool, at::optional<size_t>>(false, {});
```

Comment:
So I'm not familiar with the code, so feel free to ignore the comments, but I think you can just write:

```
return std::make_tuple(false, at::nullopt_t);
```

Reply:
Cool, good catch.